src/Connection/Connection.php

```diff
@@ -146,10 +148,11 @@ private function noReplyWait(): void
 
     /**
      * @param MessageInterface $message
-     * @return array
+     * @param bool $raw
+     * @return ResponseInterface|Cursor
```

Comment:
This should include void IMO, since if noReply, nothing is returned

Reply:
The IDE will not understand the `@return void` because the avoid means the whole method will never return anything.